### DISCUSSION TITLE
Добро пожаловать в Обсуждения (Discussions)

### DISCUSSION THREAD

**Discussion by ademaro** [makehtml/python-pikabu-tutorial#2] [Announcements]
## 👋 Добро пожаловать!
  Мы будем использовать обсуждения как место для общения с другими членами нашей команды. Я надеюсь, что вы:
   * Будете активно задавать вопросы, которые вас интересуют.
   * Делиться идеями.
   * Общаться с другими членами команды.
   * Будьте максимально открыты, помните, что общий успех зависит от нашего взаимодействия 💪.

Чтобы начать, оставьте комментарий ниже, представив себя и расскажите о том, что вы делаете в этой команде и какие личные цели преследуете.

**Comment by frcjocker** [comment]
Всем привет! Меня зовут Эрдэм.
Немного знаком с CSS, html, JS, React, Java, C++, Adruino (Университет, курсы и разные приложения).
Я хочу научиться писать на Python и Flask, участвовать в интересных проектах и осваивать новые знания!
